tools/rpkg/R/Result.R

```diff
@@ -1,10 +1,3 @@
-#' @title DuckDB Result Set
-#' @description  TBD
-#'
-#' @param connection FIXME
-#' @param stmt_lst FIXME
-#' @export
-#' @name duckdb_result
```

Comment:
Was this intended?

Reply:
This results in 

```r
checking for missing documentation entries ... WARNING
  Undocumented S4 methods:
    generic 'dbBind' and siglist 'duckdb_result'
    generic 'dbClearResult' and siglist 'duckdb_result'
    generic 'dbColumnInfo' and siglist 'duckdb_result'
    generic 'dbFetch' and siglist 'duckdb_result'
    generic 'dbGetInfo' and siglist 'duckdb_result'
    generic 'dbGetRowCount' and siglist 'duckdb_result'
    generic 'dbGetRowsAffected' and siglist 'duckdb_result'
    generic 'dbGetStatement' and siglist 'duckdb_result'
    generic 'dbHasCompleted' and siglist 'duckdb_result'
    generic 'dbIsValid' and siglist 'duckdb_result'
    generic 'show' and siglist 'duckdb_result'
```

`duckdb_result()` needs to be exported and documented.

I did so now. You might want to document the two arguments of `duckdb_result()` and have a look at the failing tests. These look like true positives on the first look but I might be wrong.